itest/lnd_multi-hop_test.go

```diff
@@ -2704,7 +2708,7 @@ func runExtraPreimageFromLocalCommit(ht *lntest.HarnessTest,
 		ht.AssertOutpointInMempool(htlcOutpoint)
 
 		// Mine a block to confirm Carol's direct spend tx.
-		ht.MineBlocks(1)
+		ht.MineBlocksAndAssertNumTxes(1, 1)
```

Comment:
I think we need a Neutrino workaround here as well? See https://github.com/lightningnetwork/lnd/actions/runs/9794675705/job/27045198822
Line number doesn't match, but looks to be the same issue.

Reply:
you mean this one?
```
harness_assertion.go:791: 
        	Error Trace:	/home/runner/work/lnd/lnd/lntest/harness_assertion.go:791
        	            				/home/runner/work/lnd/lnd/lntest/harness_assertion.go:805
        	            				/home/runner/work/lnd/lnd/itest/lnd_funding_test.go:1352
        	            				/home/runner/work/lnd/lnd/lntest/harness.go:391
        	            				/home/runner/work/lnd/lnd/itest/lnd_test.go:139
        	Error:      	Received unexpected error:
        	            	carol: assert num of UTXOs failed: want 1, got: 0, total: 0, previously had: 0
        	Test:       	TestLightningNetworkDaemon/tranche01/29-of-156/neutrino/open_channel_with_unstable_utxos
        	Messages:   	timeout waiting for UTXOs
    harness.go:445: finished test: open_channel_with_unstable_utxos, start height=524, end height=528, mined blocks=4
    harness.go:451: test failed, skipped cleanup
```

Reply:
This one:
```
 === RUN   TestLightningNetworkDaemon/tranche06/132-of-156/neutrino/watchtower/session_deletion
    miner.go:217: 
        	Error Trace:	/home/runner/work/lnd/lnd/lntest/miner/miner.go:217
        	            				/home/runner/work/lnd/lnd/lntest/harness_miner.go:229
        	            				/home/runner/work/lnd/lnd/itest/lnd_multi-hop_test.go:1218
        	            				/home/runner/work/lnd/lnd/itest/lnd_multi-hop_test.go:152
        	Error:      	Received unexpected error:
        	            	want 1, got 0 in mempool: []
        	Test:       	TestLightningNetworkDaemon/tranche03/74-of-156/neutrino/multi_hop_htlc_local_chain_claim/zeroconf=true/committype=SIMPLE_TAPROOT
        	Messages:   	assert tx in mempool timeout
```

But the one you posted looks similar too.

Reply:
I think potentially also the call to `walletUtxo := ht.AssertNumUTXOsUnconfirmed(carol, 1)[0]` in `testChannelFundingWithUnstableUtxos`

```
    harness_assertion.go:791: 
        	Error Trace:	/home/runner/work/lnd/lnd/lntest/harness_assertion.go:791
        	            				/home/runner/work/lnd/lnd/lntest/harness_assertion.go:805
        	            				/home/runner/work/lnd/lnd/itest/lnd_funding_test.go:1352
        	            				/home/runner/work/lnd/lnd/lntest/harness.go:386
        	            				/home/runner/work/lnd/lnd/itest/lnd_test.go:139
        	Error:      	Received unexpected error:
        	            	carol: assert num of UTXOs failed: want 1, got: 0, total: 0, previously had: 0
        	Test:       	TestLightningNetworkDaemon/tranche01/29-of-156/neutrino/open_channel_with_unstable_utxos
        	Messages:   	timeout waiting for UTXOs
    harness.go:440: finished test: open_channel_with_unstable_utxos, start height=524, end height=528, mined blocks=4
    harness.go:446: test failed, skipped cleanup
    lnd_payment_test.go:487: 	Benchmark info: Elapsed time:  2.340022511s
    lnd_payment_test.go:488: 	Benchmark info: TPS:  206.40827074505012
```

Reply:
cool fixed this test in a follow-up commit.